packages/react/src/LensProvider.tsx

```diff
@@ -1,5 +1,5 @@
 import { invariant } from '@lens-protocol/shared-kernel';
-import { ReactNode, useEffect, useRef, useState } from 'react';
+import React, { useEffect, useRef, useState } from 'react';
```

Comment:
What complained about this?

Reply:
The nextjs app build... was having a few issues with it. This is a quick workaround.

In Next.js `React` is a global and is not needed to be imported, I believe that would be the cause of this issue.

This PR is a working POC, if we decide to go ahead with this approach, I will come back and iron out some of these quirks.